package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remoteoss/json-schema-form",
-  "version": "0.11.10-beta.0",
+  "version": "0.11.11-dev.20250220164730",
```

Comment:
Hi! I'm doing the review now, it includes testing this PR against our internal E2E tests, therefore this new commit. Don't worry, I'll revert it once we are done.